roles/install_nextcloud/tasks/tls_signed.yml

```diff
@@ -29,28 +29,30 @@
     dest: "{{ nextcloud_tls_cert_file }}"
     owner: "{{ nextcloud_websrv_user }}"
     group: "{{ nextcloud_websrv_group }}"
-    mode: "0640"
+    mode: u=rw,g=r,o=-
   vars:
     input_files: ["{{ nextcloud_tls_src_cert }}", "{{ nextcloud_tls_src_chain }}"]
   when:
     - nextcloud_tls_src_chain is defined
     - nextcloud_websrv in ["nginx"]
 
-- name: "[SIGNED TLS] - Key is copied to the host"
+- name: "[SIGNED TLS] - Copy certificate chain file for apache2 to the host"
   ansible.builtin.copy:
-    dest: "{{ nextcloud_tls_cert_key_file }}"
-    src: "{{ nextcloud_tls_src_cert_key }}"
+    dest: "{{ nextcloud_tls_chain_file }}"
+    src: "{{ nextcloud_tls_src_chain }}"
     owner: "{{ nextcloud_websrv_user }}"
     group: "{{ nextcloud_websrv_group }}"
-    mode: "0400"
+    mode: u=rw,g=r,o=-
```

Comment:
Have you changed it from 0400 to 0640 on purpose here?

Reply:
Ouch, it should be "0400" for the key! You are right! Since I've signed-off your proposal for the mode above, should I change this or is it better you propose the change?